### ISSUE TITLE
(2.6.0) can't manage app in work profile

### ISSUE THREAD

**Issue by NSTAdventure** [MuntashirAkon/AppManager#505]
- [x] I know what my device, OS and App Manager versions are
- [x] I know how to take logs
- [x] I know how to reproduce the issue which may not be specific to my device

**Describe the bug**
Tried to manage work profile app will result AM crash

**To Reproduce**
Steps to reproduce the behaviour:
1. Open AM With ADB Over TCP Mode
2. Click on app choose work profile
3. Try to disable or enable
4. AM crash

**Expected behavior**
No crash

**Screenshots**

**Crash logs**
```
java.lang.SecurityException: Shell cannot change component state for com.oppo.ota/null to 1
    at android.os.Parcel.createException(Parcel.java:2087)
    at android.os.Parcel.readException(Parcel.java:2055)
    at android.os.Parcel.readException(Parcel.java:2003)
    at android.content.pm.IPackageManager$Stub$Proxy.setApplicationEnabledSetting(IPackageManager.java:7651)
    at io.github.muntashirakon.AppManager.servermanager.PackageManagerCompat.setApplicationEnabledSetting(PackageManagerCompat.java:106)
    at io.github.muntashirakon.AppManager.details.info.AppInfoFragment.lambda$setHorizontalActions$67$AppInfoFragment(AppInfoFragment.java:925)
    at io.github.muntashirakon.AppManager.details.info.-$$Lambda$AppInfoFragment$nS4ywqoiA_G4QduKEJU9qGxhfuw.onClick(Unknown Source:2)
    at android.view.View.performClick(View.java:7394)
    at android.view.View.performClickInternal(View.java:7348)
    at android.view.View.access$3900(View.java:827)
    at android.view.View$PerformClick.run(View.java:28293)
    at android.os.Handler.handleCallback(Handler.java:899)
    at android.os.Handler.dispatchMessage(Handler.java:100)
    at android.os.Looper.loop(Looper.java:238)
    at android.app.ActivityThread.main(ActivityThread.java:7853)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:984)
 Caused by: android.os.RemoteException: Remote stack trace:
	at com.android.server.pm.PackageManagerService.setEnabledSetting(PackageManagerService.java:23695)
	at com.android.server.pm.PackageManagerService.setApplicationEnabledSetting(PackageManagerService.java:23564)
	at com.android.server.pm.OppoPackageManagerService.setApplicationEnabledSetting(OppoPackageManagerService.java:540)
	at android.content.pm.IPackageManager$Stub.onTransact(IPackageManager.java:3638)
	at com.android.server.pm.PackageManagerService.onTransact(PackageManagerService.java:4883)


Device Info:
App version: 2.6.0
App version code: 385
Android build version: 1608537052
Android release version: 10
Android SDK version: 29
Android build ID: RMX2030EX_11_C.67
Device brand: realme
Device manufacturer: realme
Device name: RMX2030
Device model: RMX2030
Device product name: RMX2030
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: id-ID
In-App Language: auto
Mode: adb
```

**Device info**
 - Device: Realme 5i
 - OS Version: Android Q
 - App Manager Version: 2.6.0
 - Mode: ADB over TCP

**Additional context**
Not tested on 2.6.1 and 2.6.2 yet
For 2.6.1 i will never able to test since AM have ADB Over TCP Issues
For 2.6.2 Waiting Until It Released On F-Droid.org I dont want get bothered reinstall this app with different signature

**Comment by MuntashirAkon** [comment]
This isn't reproducible on my device running Android 11 with Insular on App Manager v2.6.2. v2.6.0 is unsupported, so wait for v2.6.2 first.

**Comment by MuntashirAkon** [comment]
> For 2.6.1 i will never able to test since AM have ADB Over TCP Issues

This can be fixed very easily using v2.6.0. Set mode of operations to `auto`, the update to v2.6.1 and set mode of operations to ADB again.

**Comment by NSTAdventure** [comment]
> This isn't reproducible on my device running Android 11 with Insular on App Manager v2.6.2. v2.6.0 is unsupported, so wait for v2.6.2 first.

Android 11 

I said android 10
Idk about 11 sir 🥴
